metadata_service/proxy/neo4j_proxy.py

```diff
@@ -1242,8 +1233,7 @@ def get_dashboard(self,
                                                      )
         dashboard_record = self._execute_cypher_query(statement=get_dashboard_detail_query,
                                                       param_dict={'query_key': id,
-                                                                  'tag_normal_type': 'default',
-                                                                  'tag_badge_type': 'badge'}).single()
```

Comment:
so we update the query before but forget to delete this param?

Reply:
yeah...